ambry-api/src/main/java/com.github.ambry/config/HelixPropertyStoreConfig.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.config;
+
+/**
+ * The config that is needed to operate a {@code HelixPropertyStore}.
+ */
+public class HelixPropertyStoreConfig {
+  public static final String HELIX_PROPERTY_STORE_PREFIX = "helix.property.store.";
+
+  /**
+   * Timeo in ms to time out a connection to ZooKeeper server.
+   */
+  @Config(HELIX_PROPERTY_STORE_PREFIX + "zk.client.connection.timeout.ms")
+  @Default("20000")
+  public final int zkClientConnectionTimeoutMs;
+
+  /**
+   * Time in ms defines disconnection tolerance by a session. I.e., if reconnected within this time, it will
+   * be considered as the same session.
+   */
+  @Config(HELIX_PROPERTY_STORE_PREFIX + "zk.client.session.timeout.ms")
+  @Default("20000")
+  public final int zkClientSessionTimeoutMs;
+
+  /**
+   * The ZooKeeper server address.
+   */
+  @Config(HELIX_PROPERTY_STORE_PREFIX + "zk.client.connect.string")
+  @Default("")
+  public final String zkClientConnectString;
```

Comment:
So far there is no problem to simply use the connect strings from the `ClusterMapConfig`, just in theory this is not necessarily the same as the `ClusterMapConfig`. For example, for the purpose of balancing the zk load, one zk is used for cluster management, and one zk is used for account service storage and notifications.